packages/@unimodules/core/android/src/main/java/org/unimodules/core/interfaces/ComponentLifecycleListener.java

```diff
@@ -0,0 +1,15 @@
+package org.unimodules.core.interfaces;
+
+import org.unimodules.core.ModuleRegistry;
+
+public interface ComponentLifecycleListener {
```

Comment:
I think `RegistryLifecycleListener` would be a better name (I think in this context it could be mistaken for some `React.Component`). @tsapeta?

Reply:
yeah, I'd call it `ModuleRegistryLifecycleListener`